exporter/exporterhelper/internal/queuebatch/queue_batch_test.go

```diff
@@ -449,40 +449,6 @@ func TestQueueBatch_BatchBlocking(t *testing.T) {
 	require.NoError(t, qb.Shutdown(context.Background()))
 }
 
-// Validate that the batch is cancelled once the first request in the request is cancelled
```

Comment:
I agree that this test should be removed. It does not make sense, IMO, to cancel a whole batch because one of the inputs timed out, for example.